src/components/storefront/Cart/Layout/CartSkeleton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { HStack, VStack, Skeleton, SkeletonText } from '@chakra-ui/react';
+
+const CartItemsSkeleton = ({ numItems = 3 }) => {
```

Comment:
small nit: I have 2 items in my cart but the skeleton shows the default value 3, not sure if you're passing in the props here?

![image](https://user-images.githubusercontent.com/24748963/116008046-b69e0a00-a5e0-11eb-9fe3-2d8d87a08fdc.png)
![image](https://user-images.githubusercontent.com/24748963/116008120-f664f180-a5e0-11eb-9584-1ed0576adc6d.png)

Reply:
3 is hardcoded - there's no way to no the # of items in the cart, otherwise the page would already be loaded! i just put 3 cuz I thought it looks nice enough

Reply:
oh that makes sense LOL